MomentumDiscordBot/Commands/Admin/AdminMentionRolesModule.cs

```diff
@@ -47,10 +46,9 @@ public async Task CreateMentionRoleAsync(CommandContext context, DiscordRole rol
             await ReactionBasedRoleService.SendRoleEmbed(role);
         }
 
-        [Command("remove")]
-        [Aliases("delete", "del", "rem")]
-        [Description("Removes a role for mentioning, and removes it from the react channel")]
-        public async Task RemoveMentionRoleAsync(CommandContext context, DiscordRole role)
+        [SlashCommand("remove", "Removes a role for mentioning, and removes it from the react channel")]
+        //[Aliases("delete", "del", "rem")]
```

Comment:
I agree with what you said about aliases being unnecessary, you can remove this comment and all other similar ones.